recursive/recursive_searching.js

```diff
@@ -20,6 +37,13 @@ const binary_search = (arr, target, start, end) => {}
  * descending order. You can implement this function 
  * either recursively or iteratively
  */
-const agnostic_binary_search = (arr, target) => {}
+const agnostic_binary_search = (arr, target) => {
+    for(let i = 0; i < arr.length; i++){
```

Comment:
Why are you going through the elements with a for loop instead of something similar to the binary_search function?